flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/common/CommonPhysicalLookupJoin.scala

```diff
@@ -109,7 +109,7 @@ abstract class CommonPhysicalLookupJoin(
     cluster.getRexBuilder,
     input.getRowType,
     calcOnTemporalTable,
-    allLookupKeys.keys.toList.sorted.toArray,
+    allLookupKeys.values.toList,
```

Comment:
minor: could you help update the comment to 
` remaining condition used to filter the joined records (left input record X lookup-ed records)`